src/index.html

```diff
@@ -11,6 +11,45 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      class="search-bar"
+      method="post"
+      data-qa="big"
+    >
+      <img
+        class="search-icon"
+        src="images/Search.svg"
+        alt="search-icon"
+      >
+
+      <input
+        type="text"
+        placeholder="Try &quot;Los Angeles&quot;"
+        class="search-bar__input"
+        name="search-bar"
+        data-qa="keypress"
+      >
```

Comment:
The closing tag for the input field is missing. Although it's optional in HTML5, including it can help avoid potential issues in older browsers.